rest-service/manager_rest/configure_manager.py

```diff
@@ -623,8 +693,9 @@ def _wait_for_rabbitmq(address):
 
     config.instance.load_configuration(from_db=False)
     with setup_flask_app().app_context():
-        config.instance.load_from_db(session=db.session)
         user_config = _load_user_config(args.config_file_path)
+        populate_config_in_db(user_config)
```

Comment:
move to inside `configure` I think, that's where all functions on this level are called from. Up here, at the top level, it only makes sense to place things that can be called separately, like the admin token thing